packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -399,6 +403,14 @@ class RenderUiKitView extends RenderBox {
     _lastPointerDownEvent = null;
   }
 
+  @override	
+  void describeSemanticsConfiguration (SemanticsConfiguration config) {	
```

Comment:
nit: you'll have to remove the whitespace at the and of these lines to make the analyzer happy on Cirrus.

Reply:
done